.github/workflows/ci.yml

```
@@ -310,3 +318,20 @@ jobs:
       uses: mxschmitt/action-tmate@v3
       # When debugging increase to a suitable value!
       timeout-minutes: ${{ github.event.pull_request && 1 || 20 }}
+
+    - name: Prepare code coverage report
+      # if: ${{ success() && (matrix.coverage == true) }}
+      run: |
+        # Create lcov report
+        # capture coverage info
+        vers="${CC#*-}"
+        lcov --directory . --capture --output-file coverage.info --gcov-tool /usr/bin/gcov-${vers}
+        lcov --remove coverage.info '/usr/*' "${HOME}"'/.cache/*' "$GITHUB_WORKSPACE"'/test/*' --output-file coverage.info
```

```suggestion
        lcov --remove coverage.info '/usr/*' "${HOME}"'/.cache/*' ${{ github.workspace }}'/llvm-project/*' --output-file coverage.info
```

We probably need something like that...